tests/Windows.Win32.Tests/assets/suggestedRemappingsAllowList.rsp

```diff
@@ -233,4 +233,6 @@ _WS_SERVICE_HOST=WS_SERVICE_HOST
 _WS_SERVICE_PROXY=WS_SERVICE_PROXY
 _WS_METADATA=WS_METADATA
 _WS_POLICY=WS_POLICY
-_MDL=MDL
\ No newline at end of file
+_MDL=MDL
+ABI::Windows::Foundation::Collections::IPropertySet=__PropertySet__
+ABI::Windows::Storage::Streams::IPropertySetSerializer=__IPropertySetSerializer__
```

Comment:
What are these changes for?

Reply:
Standard exclusion for noisy suggestions.